.github/actions/test_sqllogic_standalone_linux/action.yml

```diff
@@ -26,20 +26,14 @@ runs:
         profile: ${{ inputs.profile }}
         sha: ${{ github.sha }}
         target: ${{ inputs.target }}
-
-    - name: Run sqllogic Tests with Standalone mode
+    - name: Setup Build Tool
+      uses: ./.github/actions/setup_build_tool
+      with:
+        bypass_env_vars: RUSTFLAGS,RUSTDOCFLAGS,RUST_TEST_THREADS,RUST_LOG,RUST_BACKTRACE
+    - name:  Run sqllogic tests with Standalone mode
       shell: bash
+      run: cargo run -p sqllogictests -- --skip_dir cluster
```

Comment:
I didn't follow the handler and suit separately, maybe we can do that later? cc @Xuanwo